devicon.json

```
@@ -2525,6 +2525,24 @@
           }
         ]
     },
+    {
+        "name": "sqlserver",
```

I was wondering which name is sufficient for this icon? Some ideas:
- mssql
- mssqlserver
- microsoftsql
- microsoftsqlserver

```suggestion
        "name": "mssqlserver",
```

@Thomas-Boi what do you think?

---

Microsoft SQL Server is the full name

---

I think we should use the `microsoftsqlserver` since the full name is Microsoft SQL Server.

I wonder if we should consider adding hyphens in our names from now on. I'll open a discussion to discuss this.

---

I'm fine with `microsoftsqlserver`, would you mind updating it @EnisMulic?